lib/zstd/Makefile

```diff
@@ -0,0 +1,19 @@
+zstd_DIR	= $(NXDK_DIR)/lib/zstd/zstd
+zstd_SRCS	= $(sort $(wildcard $(zstd_DIR)/lib/common/*.c))
+zstd_SRCS	+= $(sort $(wildcard $(zstd_DIR)/lib/compress/*.c))
+zstd_SRCS	+= $(sort $(wildcard $(zstd_DIR)/lib/decompress/*.c))
+zstd_SRCS	+= $(sort $(wildcard $(zstd_DIR)/lib/dictBuilder/*.c))
+
+zstd_OBJS = $(addsuffix .obj, $(basename $(zstd_SRCS)))
+
+NXDK_CFLAGS += -I $(zstd_DIR)/lib -D_byteswap_ulong=__builtin_bswap32 -D_byteswap_uint64=__builtin_bswap64
```

Comment:
`_byteswap_ulong` and `_byteswap_uint64` are regular functions on Windows, so I'd recommend putting their declarations in `stdlib_ext_.h` and implement them in `stdlib_ext_.c` to avoid introducing additional compiler parameters. You can still use the builtins for the implementation, of course.

Reply:
You moved the definitions into the header, but I meant that they should be actual functions, like they are on Windows.